File: src/java/src/main/java/org/elastos/hive/vault/database/InsertResult.java
package org.elastos.hive.vault.database;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class InsertResult {
	@SerializedName("acknowledged")
	private Boolean acknowledged;

	@SerializedName("inserted_ids")
	private List<String> insertedIds;

	public Boolean getAcknowledged() {
		return acknowledged;
	}

	public List<String> getInsertedIds() {
		return insertedIds;
	}
}

File: src/restclient/database/insertresult.ts
export class InsertResult {
    private acknowledged: boolean;
    private inserted_ids: string[];

    public isAcknowledge(): boolean {
        return this.acknowledged;
    }

    public getIntertedIds(): string[] {
        return this.inserted_ids;
    }

    public setAcknowledge(isAcknowledge: boolean): void {
        this.acknowledged = isAcknowledge;
    }

    public setInsertedIds(insertedIds: string[]): void {
        this.inserted_ids = insertedIds;
    }
}